### PR TITLE
Importing models and animations

### PR DESCRIPTION
**Pull request by dschonholtz**
* Loads a 3d model 
* Runs an animation on that 3d model
* Adds movement and rotation to that 3d model to make the animation look like it does something

That last part is still a little hacky as I'm doing the rotation manually instead of with trig because I was lazy and did this in just an hour or two. 
On top of that, when we transition to say 3.1 radians to 0.1 radians it rotates all the way around instead of rotating the shorter distance so sometimes it will do pirouettes which isn't great. 
I still think it is worth merging this into master as from here on out I think we will be building on this with more markers, models, and animations.

### CODE REVIEW

**Comment by dschonholtz** [comment]
Ok I just ran this again and the rotation is REALLY hacky and maybe it is worth taking it out even here but I'll leave it in for now so you guys can get a giggle.

File: src/index.js [hunk 1]
```diff
@@ -80,4 +98,52 @@ function render() {
...
+    zv += 0.001;
+  }
+  if (model.position.x > 0.4) {
+    xv -= 0.001;
+  }
+  if (model.position.z > 0.4) {
+    zv -= 0.001;
+  }
+
+  xv += Math.random() * (0.001 - -0.001) + -0.001;
```
Comment (DarkAce65) [inline src/index.js]: Mind simplifying the math a bit here? Maybe something like this if you want to maintain the `0.001`:
```suggestion
  xv += Math.random() * (0.001 * 2) - 0.001;
```

File: src/index.js [hunk 2]
```diff
@@ -80,4 +98,52 @@ function render() {
...
+  zv += Math.random() * (0.001 - -0.001) + -0.001;
+
+  // This is really sad but I don't remember how to do this with trig and this works:
+  let rotation = (Math.abs(xv) / Math.abs(zv)) * 1.57;
```
Comment (DarkAce65) [inline src/index.js]: Can you use `Math.PI` for the pi related constants?

**Review by DarkAce65 — COMMENTED** [review]
Looks good to me! Just left a few small comments

File: src/index.js (hunk 1 above)
Comment (dschonholtz) [reply src/index.js]: Done. Was just trying to make the range super clear but if you have half a brain it's pretty clear either way.

File: src/index.js [hunk 3]
```diff
@@ -125,22 +123,22 @@ function updatePosition() {
...
   } else if (xv < 0 && zv < 0) {
-    // pi -> 3/4 pi
-    rotation += 3.14;
+    // pi -> 3/2 pi
+    rotation += Math.PI;
   } else {
-    // 3/4 pi to 2 pi
-    rotation += 4.71;
+    // 3/2 pi to 2 pi
+    rotation += Math.PI * 0.75;
```
Comment (Abbondanzo) [inline src/index.js:141]: Do each of these comments refer to the ranges in question or the arithmetic?

**Pull request merged**